projects/litentry/README.md

```diff
@@ -2,12 +2,16 @@
 
 ## Project Introduction
 
-Litentry is a cross-chance Decentralized Identity (DID) Aggregator that enables the linking of user identities among multiple decentralized networks. Built on the Substrate framework and featuring a DID indexing protocol, Litentry provides a decentralized, verifiable identity aggregation service that removes the redundancy of code and the hassles involved in resolving agnostic DID mechanisms. Litentry’s DID Aggregator is compatible with all DID standards, and powered by a reliable DID data interface. Everyone can build and submit DID methods to Litentry, making identity data easily accessible in the Web3.0.
+Litentry is a cross-chain Decentralized Identity (DID) Aggregator that enables the linking of user identities among multiple decentralized networks. Built on the Substrate framework and featuring a DID indexing protocol, Litentry provides a decentralized, verifiable identity aggregation service that removes the redundancy of code and the hassles involved in resolving agnostic DID mechanisms. Litentry’s DID Aggregator is compatible with all DID standards, and powered by a reliable DID data interface. Everyone can build and submit DID methods to Litentry, making identity data easily accessible in the Web3.0.
 
 ## Wasm contract technical support
 
-Coming soon.
+Litentry smart contract is built on Patract's Redspot tool, allowing DApp developers to integrate cross-chain identities and data on Litentry easily.
+Furthermore, developers can perform identity calculations for different scenarios based on their algorithmic logic and customize their own user reputation/rating system.
+Github Link: https://github.com/litentry/litentry-contract.
 
 ## Developer activity
 
-Coming soon.
+Litentry has extended Europa, the contract runtime sandbox environment provided by Patract, to increase the accessibility of Litentry Pallet data, allowing direct simulation of interactions with Litentry nodes.
```

Comment:
This part is mainly describe what kind of technical activities your will have for developers in the future. If you don't have any plans，just delete this part.
(This sentence should be transferred to ##Wasm contract technical support)